### PR TITLE
Add tensap

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/tensap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  name: tensap
+  version: {{ version }}
+
+source:
+  #url: https://github.com/anthony-nouy/tensap/archive/v{{ version }}.tar.gz
+  #sha256: e0828ad9b2a330d996b2ce4a445980dd7bc22e79573abb7b36ff0920779b8a9d
+
+  # we need a few commits after v1.0 tag for the setup.py
+  git_url: https://github.com/anthony-nouy/tensap
+  git_rev: a5d26f4de04f2868f03d5a7932ca64d9d127c93a
```
Comment (scopatz) [inline recipes/tensap/meta.yaml:13]: Please build from source, not git

Comment (jschueller) [reply recipes/tensap/meta.yaml:13]: @scopatz I would, but as mentioned in the comment above, we need a few more commits past the v1.0 tag that added support for setup.py

Comment (scopatz) [reply recipes/tensap/meta.yaml]: Can you do this for an old release for now? Or add a patch file here?

File: recipes/tensap/setup.patch [hunk 2]
```diff
@@ -0,0 +1,825 @@
+From a5d26f4de04f2868f03d5a7932ca64d9d127c93a Mon Sep 17 00:00:00 2001
```
Comment (scopatz) [inline recipes/tensap/setup.patch:1]: This file isn't being used - is that because there is a new release?

Comment (jschueller) [reply recipes/tensap/setup.patch:1]: oh, my mistake

File: recipes/tensap/meta.yaml (hunk 1 above)
Comment (jschueller) [reply recipes/tensap/meta.yaml]: using v1.1

**Pull request closed without merging**